src/components/App.js

```diff
@@ -57,8 +79,14 @@ function App() {
             document.title = "Sketchify - Untitled";
         } else {
             document.title = "Sketchify - " + title;
-            localStorage.setItem("codepen-clone-title", title);
+            localStorage.setItem("sketchify-title", title);
```

Comment:
Please change the spelling here to keep the code consistent.

Reply:
Sorry I didn't get what you are trying to say...?

Reply:
Yes

Reply:
@s-katte The local storage keys are all set with the "sketchify-title-" prefix; that's why I did this.
Should I still make changes?

Reply:
okay yes, so sorry.